iso_templates/initrd_init_template

```diff
@@ -104,15 +128,46 @@ mount_root() {
     fi
 }
 
-# Finds the installer media and calls mount_root if successful
-find_and_mount_installer() {
+wait_for_timeout() {
+    local n="$1"
+    echo_tty_kmsg
+    echo_tty_kmsg
+    while [ "$n" -gt 0 ] ; do
+        printf "Continue booting in [%2d] sec(s)...\r" "$n" > "${TTY}"
+        sleep 1
+        n=`expr "$n" - 1`
+    done
+}
+
+check_iso_verify() {
+    local ret="$1"
+    local msg="$2"
+    [ "$ret" -ne 0 ] && { echo_tty_kmsg "[${FAILURE} FAIL ${NORMAL}]"; shell_trap "$msg"; }
+    [ "$ret" -eq 0 ] && { echo_tty_kmsg "[${SUCCESS}  OK  ${NORMAL}] $msg"; wait_for_timeout 10; }
+}
+
+# Check ISO media integrity
+verify_media() {
+    local installer="$1"
+    local need="ISO integrity check"
+
+    if [ -n "${installer}" ]; then
+        checkisomd5 --verbose ${installer} |& tee -a "${TTY}"
```

Comment:
Sorry, one other fix
```suggestion
        checkisomd5 --verbose ${installer} > "${TTY}"
```
QEMU was getting the output twice before this change.

Reply:
Yes , I missed it. Will fix , thanks. I saw something to fix too. Will push them together

Reply:
Done